action/protocol/poll/staking_committee.go

```diff
@@ -139,6 +142,7 @@ func (sc *stakingCommittee) CreateGenesisStates(ctx context.Context, sm protocol
 		GetBlockHash: func(height uint64) (hash.Hash256, error) {
 			return hash.ZeroHash256, nil
 		},
+		GetBlockTime: sc.getBlockTime,
```

Comment:
it seems it's only used for genesis block, so we can know Shanghai is not enabled, let's figure out a way w/o needing to pass in a real func